man/man7/zfsprops-macos.7

```diff
@@ -0,0 +1,94 @@
+.\" file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+.\" If applicable, add the following below this CDDL HEADER, with the
+.\" fields enclosed by brackets "[]" replaced with your own identifying
+.\" information: Portions Copyright [yyyy] [name of copyright owner]
+.\"
+.\" CDDL HEADER END
+.\"
+.\"
+.\" Copyright (c) 2009 Sun Microsystems, Inc. All Rights Reserved.
+.\" Copyright 2011 Joshua M. Clulow <josh@sysmgr.org>
+.\" Copyright (c) 2011, 2019 by Delphix. All rights reserved.
+.\" Copyright (c) 2011, Pawel Jakub Dawidek <pjd@FreeBSD.org>
+.\" Copyright (c) 2012, Glen Barber <gjb@FreeBSD.org>
+.\" Copyright (c) 2012, Bryan Drewery <bdrewery@FreeBSD.org>
+.\" Copyright (c) 2013, Steven Hartland <smh@FreeBSD.org>
+.\" Copyright (c) 2013 by Saso Kiselkov. All rights reserved.
+.\" Copyright (c) 2014, Joyent, Inc. All rights reserved.
+.\" Copyright (c) 2014 by Adam Stevko. All rights reserved.
+.\" Copyright (c) 2014 Integros [integros.com]
+.\" Copyright (c) 2016 Nexenta Systems, Inc. All Rights Reserved.
+.\" Copyright (c) 2014, Xin LI <delphij@FreeBSD.org>
+.\" Copyright (c) 2014-2015, The FreeBSD Foundation, All Rights Reserved.
+.\" Copyright 2019 Richard Laager. All rights reserved.
+.\" Copyright 2018 Nexenta Systems, Inc.
+.\" Copyright 2019 Joyent, Inc.
+.\" Copyright (c) 2019, Kjeld Schouten-Lebbing
+.\"
+.Dd April 29, 2021
+.Dt ZFSPROPS-MACOS 7
+.Os
+.
+.Sh NAME
+.Nm zfsprops-macos
+.Nd native properties of ZFS datasets specific to macOS
+.
+.Sh DESCRIPTION
+These properties are like the native properties described in
+.Xr zfsprops 7
+but they are specific to macOS.
+.Pp
+The following native properties can be used to change the behavior of a ZFS
+dataset.
+.Bl -tag -width ""
+.It Xo
+.Sy browse Ns = Ns Sy on Ns | Ns Sy off
+.Xc
+Equivalent to mount option
+.Sy browse/nobrowse .
+This option indicates
+that the mount point should not be visible via the GUI (i.e., appear
+on the Desktop as a separate volume).  Setting it to off will result
+in Spotlight being unavailable for the specified mount point.
+The default value is
+.Sy off .
+.It Xo
+.Sy ignoreowner Ns = Ns Sy on Ns | Ns Sy off
+.Xc
+Equivalent to mount option
+.Sy noowners .
+Ignore the ownership field for the entire volume.
+The default value is
+.Sy off .
+.It Xo
+.Sy mimic Ns = Ns Sy hfs Ns | Ns Sy apfs Ns | Ns Sy ntfs Ns | Ns Sy off
+.Xc
+Some applications check if the filesystem type is and refuse to work with ZFS.
+In this situation, this property can be enabled and the chosen filesystem type
+will be reported.
+The default value is
+.Sy off .
+.It Xo
+.Sy devdisk Ns = Ns Sy poolonly Ns | Ns Sy on Ns | Ns Sy off
+.Xc
+Create a /dev/diskX pseudo-disk for the mount, which has better compatibility
+with macOS UI.
+Due to the cost of creating a /dev/disk entry, the default value is set to
+.Sy poolonly
+as to only create an entry for the pool's root dataset.
+For lower datasets, set to
+.Sy on .
+.El
```

Comment:
Considering all these properties will be visible on all platforms, documenting them in separate file might cause confusion once user won't find their documentation.